include/libc64/sleep.h

```diff
@@ -0,0 +1,12 @@
+#ifndef LIBC64_SLEEP_H
+#define LIBC64_SLEEP_H
+
+#include "ultra64.h"
+
+void csleep(OSTime time);
+void nsleep(u32 nsec);
+void usleep(u32 usec);
+void msleep(u32 ms);
```

Comment:
For consistency reasons, shouldn't this be
```suggestion
void msleep(u32 msec);
```